sdk/nodejs/src/client.ts

```diff
@@ -69,6 +72,9 @@ export class DevCycleClient {
     }
 
     constructor(sdkKey: string, options?: DevCycleServerSDKOptions) {
+        // generate UUID for this client
+        this.clientUUID = randomUUID()
+        this.hostname = `${this.clientUUID}@${os.hostname()}`
```

Comment:
shouldnt call this hostname if its not the hostname. Maybe "user_id" or "event_identifier" or something?